frontend/src/ui/views/events/details/EventDetailsView.vue

```diff
@@ -314,14 +314,19 @@
             </li>
             <li
                 v-if="signedInUser.permissions.includes(Permission.READ_USER_DETAILS)"
-                class="context-menu-item disabled"
+                class="context-menu-item"
+                @click="eventUseCase.downloadImoList(event)"
             >
                 <i class="fa-solid fa-clipboard-user" />
                 <span>IMO Liste generieren</span>
             </li>
-            <li class="context-menu-item disabled">
+            <li
+                v-if="signedInUser.permissions.includes(Permission.READ_USER_DETAILS)"
```

Comment:
permission check not required here, as the verzehtliste only contains information readable for all signed in crew members